src/components/apps/toolbar/AppNavigation.js

```diff
@@ -71,14 +71,6 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-function getAppTypeFilters() {
-    return Object.values(appType).map((type) => {
```

Comment:
There's a GitHub Actions warning now `'appType' is defined but never used` for the `import` on line 12.

Reply:
😩 I have had enough of this so I fixed the github actions lint checking at 10 pm so that it will fail on lint issues.